examples/lm1b/configs/default.py

```diff
@@ -115,6 +115,35 @@ def get_config():
   config.seed = 0
 
   # Prompt for language model sampling.
-  config.prompts = "I love to "
+  config.prompts = 'I love to '
+
+  # Parallelism
+  config.mesh_axes = ['data', 'fsdp', 'tensor']
+  config.logical_axis_rules = [
+      ['activation_batch', ['data', 'fsdp']],
+      ['activation_length', ['data', 'fsdp']],
+      ['activation_embed', 'tensor'],
+      ['activation_mlp', 'tensor'],
+      ['activation_heads', 'tensor'],
+      ['activation_kv', 'tensor'],
+      ['activation_vocab', 'tensor'],
+      ['mlp', 'tensor'],
+      ['vocab', 'tensor'],
+      ['embed', 'fsdp'],
+      ['heads', 'tensor'],
+  ]
+  config.full_sharding = [['data', 'fsdp', 'tensor']]
```

Comment:
Why is this a list of lists and later used with * expansion? Why not simple a list and pass it to `NamedSharding` without expanding it? Same for `data_sharding` below.